data/json/monsters/nether.json

```diff
@@ -394,6 +394,48 @@
       "NOGIB"
     ]
   },
+  {
+    "id": "mon_jeweled_tortoise",
+    "type": "MONSTER",
+    "name": { "str": "Jeweled Tortoise" },
+    "description": "A flattened oval shape the size of a small car, suspended above five stubby leg-like structures that don't even connect to the main body.  The body is covered entirely with countless pentagonal diamond shapes each the size of your eye, which appear to shift colours in a constant, rhythmic fashion.  You are unable to tell if the diamonds are armor, some form of sensory organs, a weapon, or all three.",
+    "default_faction": "nether",
+    "bodytype": "spider",
+    "species": [ "NETHER" ],
+    "diff": 10,
+    "volume": "875000 ml",
+    "weight": "700 kg",
+    "hp": 350,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "U",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "armor_bash": 17,
+    "armor_cut": 18,
+    "armor_bullet": 20,
+    "armor_acid": 6,
+    "armor_fire": 6,
+    "armor_elec": 8,
+    "bleed_rate": 10,
+    "vision_day": 25,
+    "vision_night": 8,
+    "harvest": "zombie_meatslug",
```

Comment:
I wouldn't use tainted meat here. Maybe disappearing in a brilliant explosion or something like that if you don't want to get bogged down in the "what exactly are these made of" question.

Reply:
yeah I could do that. The design of the monster is “inspired” by the angels in Evangelion and they have some explosive deaths there. I’ll only be available some time after 2PM. though so It will be a while.